src/helper.cc

```diff
@@ -132,7 +132,7 @@ HelperServerBase::dropQueued()
         void *cbdata;
         if (cbdataReferenceValidDone(r->request.data, &cbdata)) {
             r->reply.result = Helper::Unknown;
-            r->request.callback(cbdata, r->reply);
+            parentHelper()->callBack(r->request.callback, cbdata, r->reply);
```

Comment:
Instead of adding a virtual parentHelper() -- a red flag for several reasons -- I would rather add a Helper::Client reference as a dropQueued() parameter.

Reply:
Done in PR229.